### PR TITLE
feat(rtc_reaplyer): add rtc_replayer (#1993)

### PR DESCRIPTION
**Pull request by taikitanaka3**
* feat(rtc_replayer): add rtc_reaplayer

Signed-off-by: taikitanaka3 <taiki.tanaka@tier4.jp>

## Description

this can be merged if there is no build error 
this will be added to debug tool
https://github.com/tier4/debug_tools/blob/ad465ef3d3e082e6a2bf42165fa6a03f4d7b4c61/analyze_without_planning.tmux#L66

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by shmpwk** [comment]
Let me confirme to make sure that this PR is only for `[beta/v0.5.2-odaiba3](https://github.com/tier4/autoware.universe/tree/beta/v0.5.2-odaiba3)` right?

**Comment by taikitanaka3** [comment]
@shmpwk 
This is independent package so this can be for any version, is there any other version to include this PR?

**Comment by shmpwk** [comment]
Ok, I found this, thank you
https://github.com/autowarefoundation/autoware.universe/pull/1993

**Review by shmpwk — APPROVED** [review]

File: planning/rtc_replayer/launch/rtc_replayer.launch.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<launch>
+  <arg name="param_path" default="$(find-pkg-share rtc_auto_mode_manager)/config/rtc_auto_mode_manager.param.yaml"/>
+
```
Comment (kosuke55) [inline planning/rtc_replayer/launch/rtc_replayer.launch.xml:3]: this param file does not seem to exist in odaiba3

Comment (kosuke55) [reply planning/rtc_replayer/launch/rtc_replayer.launch.xml:3]: this was added with https://github.com/autowarefoundation/autoware.universe/pull/1541 to main

**Pull request merged**